translations/stripes-smart-components/en.json

```diff
@@ -82,5 +82,9 @@
   "deleteTag": "Remove tag",
   "enterATag": "Enter a tag.",
   "tagsTextArea": "Tag text area",
-  "addTag": "Add"
+  "addTag": "Add",
+
+  "password-strength.weak": "Your password is too weak.",
```

Comment:
How about `passwordStrength.weak` etc to be consistent with the capitalization scheme elsewhere in this file?

Reply:
Sure. Updated